lib/cylc/scheduler.py

```diff
@@ -352,16 +354,19 @@ def configure(self):
         # Start up essential services
         self.proc_pool = SubProcPool()
         self.state_summary_mgr = StateSummaryMgr()
+        self.ws_data_mgr = WsDataMgr(self)
```

Comment:
Not initialized in `__init__`... other variables are declared as `None` in the `__init__` method first...